src/views/ProductView.vue

```diff
@@ -197,19 +197,11 @@
                 <v-sheet border>
                     <v-carousel cycle show-arrows="hover">
                         <v-carousel-item
+                            v-for="media in productVariantInfoRelevantToBuyer
```

Comment:
That looks like a misuse of v-for:
- `v-for` should be defined on the parent element
- `v-for` should have a `v-key` (https://vuejs.org/style-guide/rules-essential.html#use-keyed-v-for)

Reply:
@delvh AFAIK `v-for` does not need to be defined on the container but should be defined on a list item. That is even the case in the example you can find when following the link you provided. Also on the documentation page of the `v-carousel`, there are examples placing the `v-for` on the `v-carousel-item` (see [https://vuetifyjs.com/en/components/carousels/#model](https://vuetifyjs.com/en/components/carousels/#model)). But regarding the "keying" of elements, I agree.

Reply:
The ID of the media should be a "good" key.